extensions/amp-story/1.0/test/test-amp-story-interactive.js

```diff
@@ -284,5 +286,41 @@ describes.realWin(
 
       expect(percentages4).to.deep.equal([33, 33, 33]);
     });
+
+    it('should update store service when selecting option with click', async () => {
+      const actionsListenerSpy = env.sandbox.spy();
+      addConfigToInteractive(ampStoryInteractive);
+      ampStoryInteractive.buildCallback();
+      await ampStoryInteractive.layoutCallback();
+      storeService.subscribe(
+        StateProperty.INTERACTIVE_REACT_STATE,
+        actionsListenerSpy
+      );
+
+      await ampStoryInteractive.getOptionElements()[2].click();
+
+      expect(actionsListenerSpy).to.have.been.calledOnce;
```

Comment:
Can we do `storeService.get(state)` and assert what's stored? This will be a great test for your upcoming function that computes the state.

Reply:
Sure, commited

Reply:
This test asserts that the store notifies you when a change to the STATE happens. You already test it in `test-amp-story-store-service.js`. A bug in the store service making the `story-interactive` test file fail would be pretty unintuitive, I think we should only keep the third test you have here `'should update the store property correctly'`.
It asserts that the `interactive` component sent the proper data to the store, which is what we care about. No need to test the store's behavior.